### DISCUSSION TITLE
How to set always private

### DISCUSSION THREAD

**Discussion by pippo73** [photoprism/photoprism#3649] [Q&A]
Hello all, 
I'm new to this platform wich I find well done, but quite complex and wide :-)
I'm writing you to know if there is some way to set that all the uploaded images are flagged as private, and only the user can choose the ones who should be pubblic?
thanks for your support.

**Comment by graciousgrey** [comment]
That's not possible at the moment. We do have a feature request that is very similar to yours: 
- https://github.com/photoprism/photoprism/issues/3614

**Reply by pippo73** [reply]
It is not the same, but maybe similar for your developing flow.
I don't want to make it under review, but make them private. In order that only the user who uploaded the files can see them, and only after that decide if make the pubblic.

anyway hope to hear soon some feature like this :-)

thank you anyway for the reply
